common-custom-user-data-maven-extension/README.md

```diff
@@ -24,7 +24,8 @@ This table details the version compatibility of the Common Custom User Data Mave
 
 | Common Custom User Data Maven extension versions | Gradle Enterprise Maven extension versions |
 | ------------------------------------------------ | ------------------------------------------ |
-| `1.7+`                                            | `1.10.1+`                                  |
+| `1.8+`                                           | `1.11+`                                  |
```

Comment:
What type of error message do we get if the versions are incompatible? Will it simply not do anything and silently fail?

Reply:
It's silently ignored. We do get a 
```
TRACE: Sisu - Ignoring Plexus role: com.gradle.maven.extension.api.GradleEnterpriseListener
java.lang.TypeNotPresentException: Type com.gradle.maven.extension.api.GradleEnterpriseListener not present
...
```
exception logged at the `trace` level (the deepest) but no one will see that.

Given the choice I made to _only_ provide a `@Component` in the extension, we cannot do more than that. We are sure the components will be registered on 'SessionStarted' for instance, so we could also provide a `AbstractMavenLifecycleParticipant` that checks if the `GradleEnterpriseListener` type is present on the CCUDME realm, and log sth if that's not the case, but I think this is taking a step back (complexifying back the CCUDME).